smart_card_connector_app/src/externs/chrome_smart_card_provider_private.js

```diff
@@ -0,0 +1,20 @@
+chrome.smartCardProviderPrivate = {};
```

Comment:
Consider adding this to `third_party/closure-compiler/src_additional/chrome_extensions.js` instead.